docs/source/preview_pandas_datasets.md

```diff
@@ -0,0 +1,74 @@
+# Preview Tabular Data in Kedro-viz
```

Comment:
This content in this section is not new and it is just moved to a new page. Earlier it was a part of the 'Preview Datasets' page